cloud-consumption-api/src/main/java/com/sequenceiq/consumption/api/v1/consumption/endpoint/ConsumptionInternalEndpoint.java

```diff
@@ -57,4 +57,40 @@ ConsumptionExistenceResponse doesStorageConsumptionCollectionExist(@AccountId @Q
             @QueryParam("monitoredResourceCrn") String monitoredResourceCrn, @NotEmpty @QueryParam("storageLocation") String storageLocation,
             @ValidCrn(resource = { CrnResourceDescriptor.USER, CrnResourceDescriptor.MACHINE_USER })
             @QueryParam("initiatorUserCrn") @NotEmpty String initiatorUserCrn);
+
+    @POST
+    @Path("schedule/ebs")
```

Comment:
I think the url order is wrong, much better to ebs/schedule

Reply:
we should use one new endpoint for every future consumption registration